src/fetchMany.ts

```diff
@@ -9,10 +9,14 @@ import { getOrigins } from './utils';
 export type LinkRel = 'preconnect' | 'preload';
 
 /**
- * A function that will clenaup all resource hints
+ * A function that will cleanup all resource hints
  */
 export type CleanupResourceHints = () => void;
 
+/** No-op function generator */
+// tslint:disable-next-line: no-empty
+const nooper = (): (() => void) => (): void => {};
```

Comment:
You can use the type `CleanupResourceHints` to improve readability here

Reply:
Thanks, done.